build.gradle.kts

```
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
```

```suggestion
 * Copyright (C) 2020 Knot.x Project
```

Please fix also other files :)

---

The year in the license headers is not specified by the AL standards.
AFAIK the good practice is to leave the header as the year of the project/module inception. Which is `2019` for knotx-docker.